src/components/statements/controllers.tsx

```diff
@@ -202,6 +202,15 @@ export async function viewStatement(
     );
   }
 
+  const today = new Date();
+
+  let rangeStop;
+  if (format(startOfMonth(today), YYYMMDD) == format(startOfMonth(rangeStart), YYYMMDD)) {
+    rangeStop = format(today, "yyyy-MM-dd'T'HH:mm:ss'Z'");
+  } else {
+    rangeStop = add(rangeStart, { months: 1 });
+  }
```

Comment:
can you explain what we're trying to do here?

Reply:
thanks for explaining. just change `==` to `===`

Reply:
@schmie if you take away the `ormat`,could you use https://date-fns.org/v2.28.0/docs/isEqual to compare dates?